server/paymentProviders/stripe/virtual-cards.js

```diff
@@ -157,10 +158,10 @@ export const processAuthorization = async (stripeAuthorization, stripeEvent) =>
   const currency = stripeAuthorization.pending_request.currency.toUpperCase();
   const amount = convertToStripeAmount(currency, Math.abs(stripeAuthorization.pending_request.amount));
   const collective = virtualCard.collective;
-  const balance = await collective.getBalanceWithBlockedFundsAmount({ currency });
+  const balance = await getCurrentBalance(collective.id, host.id, currency);
```

Comment:
Could we keep `getBalanceWithBlockedFunds` as a name for the function for now?

It's generally more obvious so you don't mistake it with the normal "balance".